tests/test_telemetry.py

```diff
@@ -7,10 +7,6 @@
 
 @released(python="1.7.0", dotnet="2.12.0", java="0.108.1", nodejs="3.2.0", ruby="1.4.0", golang="1.49.0")
 @bug(context.uds_mode and context.library < "nodejs@3.7.0")
-@bug(
```

Comment:
You must keep the version where it has been fixed. Otherwise, test on old travers will fail.

Reply:
The test on old versions should now work, since they failed when `DD_INSTRUMENTATION_TELEMETRY_ENABLED=1` was used, but `DD_INSTRUMENTATION_TELEMETRY_ENABLED=true`  worked. Given that this PR uses `true` everywhere, this bug should not be triggered by the test.